### ISSUE TITLE
example glue + fvtest error

### ISSUE THREAD

**Issue by mgaudet** [eclipse/omr#317]
Feel free to close if this is user error: 

```
$ OMRGLUE=example/glue SPEC=linux_x86-64 ./configure --enable-fvtest
$ make 
...
...
StartupManagerTestExample.cpp: In member function ‘virtual bool MM_StartupManagerTestExample::parseLanguageOptions(MM_GCExtensionsBase*)’:
StartupManagerTestExample.cpp:114:16: error: ‘class MM_GCExtensionsBase’ has no member named ‘fvtest_forceScavengerBackout’
    extensions->fvtest_forceScavengerBackout &= extensions->scavengerEnabled;
                ^
StartupManagerTestExample.cpp:114:60: error: ‘class MM_GCExtensionsBase’ has no member named ‘scavengerEnabled’
    extensions->fvtest_forceScavengerBackout &= extensions->scavengerEnabled;
                                                            ^
StartupManagerTestExample.cpp:115:16: error: ‘class MM_GCExtensionsBase’ has no member named ‘fvtest_forcePoisonEvacuate’
    extensions->fvtest_forcePoisonEvacuate &= extensions->scavengerEnabled;
                ^
StartupManagerTestExample.cpp:115:58: error: ‘class MM_GCExtensionsBase’ has no member named ‘scavengerEnabled’
    extensions->fvtest_forcePoisonEvacuate &= extensions->scavengerEnabled;
                                                          ^
../../omrmakefiles/rules.mk:360: recipe for target 'StartupManagerTestExample.o' failed
make[2]: *** [StartupManagerTestExample.o] Error 1
$
```

**Comment by rwy0717** [comment]
The example glue relies on run_configure.mk and configure includes to pass the right flags through.

try `make -f run_configure.mk OMRGLUE=example/glue SPEC=linux_x86-64`

I think we should consider removing run_configure.mk from OMR in the near future.

**Comment by rwy0717** [comment]
The FV tests actually were missing an ifdef, my bad. fixed in #320.

**Comment by rwy0717** [comment]
More issues. With `./configure && make && make test`, there are five new failures. I don't expect the CI to catch these.

```
[  FAILED  ] 5 tests, listed below:
[  FAILED  ] PortMemTest.mem_test7_allocate32
[  FAILED  ] PortMemTest.mem_test8_categories
[  FAILED  ] PortDumpTest.dump_test_create_dump_with_NO_name
[  FAILED  ] PortDumpTest.dump_test_create_dump_with_name
[  FAILED  ] PortDumpTest.dump_test_create_dump_from_signal_handler
```

**Comment by charliegracie** [comment]
@mgaudet does this work for now after #320 was accepted?

**Comment by mgaudet** [comment]
Yep seems good to me. :white_check_mark:

**Comment by mgaudet** [comment]
I'll open a separate issue for the mem tests @rwy0717 pointed out.
